schemata/text.yaml

```diff
@@ -9,7 +9,13 @@ properties:
   id:
     "$ref": "base.json#/definitions/uuid"
   type:
-    "$ref": "contentblock.json#/definitions/type"
+    description: Block type
+    example: "text"
+    type: string
+    enum:
+      - text
+      - hr
+      - unknown
```

Comment:
`unknown` is a unknown block type. Allowed in schema for HTML block types html-flatten can't classify, but dropped before solving